client-tools/Shell/V8ClientConnection.h

```diff
@@ -134,6 +139,13 @@ class V8ClientConnection {
       std::unordered_map<std::string, std::string> const& headerFields,
       bool raw);
 
+#ifdef ARANGODB_ENABLE_FAILURE_TESTS
+  uint32_t requestFuzz(fuzzer::RequestFuzzer& fuzzer);
+  static constexpr uint32_t _kFuzzClosedConnectionCode = 1000;
+  static constexpr uint32_t _kFuzzNoResponseCode = 1001;
+  static constexpr uint32_t _kFuzzNotConnected = 1002;
+#endif
```

Comment:
Can be moved to the cpp file (anonymous namespace there).
Rename the constants so that the underscore is removed.

Reply:
Addressed